test/docs/debug.test.js

```diff
@@ -93,10 +93,15 @@ describe('debug: shell', function() {
   });
 
   it('should avoid sending null session option with document ops (gh-13052)', async function() {
+    const m = new mongoose.Mongoose();
+    m.set('debug', true);
```

Comment:
does this need to stay?

Reply:
Unfortunately yes. Without `m.set('debug', true)` this test was failing for me locally. Not sure why.

Reply:
i didnt see this was placed in `debug.test.js`, so i guess it is correct to have it

though from what i can tell, this test is not about testing debug. also removing that "debug true" statement locally does not fail the test for me

Reply:
The test is using the debug mode setting to ensure that we're not sending `session: null` to the server. I made the test use debug a bit more explicitly.